### ISSUE TITLE
Issue on reproduce MuJoCo results

### ISSUE THREAD

**Issue by Ending2015a** [Div99/IQ-Learn#2]
Hello! Could you provide the hyperparameters and the number of training steps of each MuJoCo env to reproduce the Table5 results?(Appendix D.2 in the original paper)
I've tried the `iq_learn/scripts/run_mujoco.sh` script to train on `Ant-v2` for \~300k steps with default hyperparameters and 10 expert trajectories. But only got the eval returns around 3000~4000. The `eval/episode_reward` shows 3301.59521 and the `best_returns` is 4275.31665. thank you!

**Comment by Div-Infinity** [comment]
Hi, thanks for the interest in this work. We reliably reach ~4700 as reported in the paper on Ant-v2 with 10 experts over three different seeds. See the chart below: 

Interactive W&B Link: [Wandb Chart](https://wandb.ai/implicit-irl/Ant-v2_Vis/reports/Ant-v2-with-10-expert-demos--VmlldzoxNDQyNjky?accessToken=ou8ohusaykxyvl4bqj9mk8xp985imvpd1zt9i8m6hgd9kmzfx31nek34sxgpsub)

![Ant-v2 with 10 experts](https://user-images.githubusercontent.com/22247549/149603431-dbfcbbf2-2a35-4448-852b-1c6ec34250c1.png)

**Comment by Div-Infinity** [comment]
These are the parameters used: 
`agent=sac agent.actor_lr=3e-05 agent.critic_lr=0.0003 agent.init_temperature=0.001 agent.learnable_temperature=False env=ant env.learn_steps=1e6 env.demos=10 method.loss=value method.regularize=True num_actor_updates=1 num_seed_steps=0 train.batch=256 train.soft_update=True train.use_target=True seed=2`

**Comment by Div-Infinity** [comment]
Let me know if there are any issues with reproducing the other MuJoCo envs. Happy to help!

**Comment by Ending2015a** [comment]
> These are the parameters used: `agent=sac agent.actor_lr=3e-05 agent.critic_lr=0.0003 agent.init_temperature=0.001 agent.learnable_temperature=False env=ant env.learn_steps=1e6 env.demos=10 method.loss=value method.regularize=True num_actor_updates=1 num_seed_steps=0 train.batch=256 train.soft_update=True train.use_target=True seed=2`

I reproduced the results with these parameters. thank you!

**Comment by robfiras** [comment]
> These are the parameters used: `agent=sac agent.actor_lr=3e-05 agent.critic_lr=0.0003 agent.init_temperature=0.001 agent.learnable_temperature=False env=ant env.learn_steps=1e6 env.demos=10 method.loss=value method.regularize=True num_actor_updates=1 num_seed_steps=0 train.batch=256 train.soft_update=True train.use_target=True seed=2`

Did you use these hyperparameters in the other mujoco environments as well? If not, it would be great if you could share them as well. Thanks!
